server/services/sample_data/flights_saved_objects.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { i18n } from '@osd/i18n';
+import { ConfigSchema } from '../../../common/config';
+
+const layerList = (config: ConfigSchema) => [
+  {
+    name: 'Default map',
+    description: '',
+    type: 'opensearch_vector_tile_map',
+    id: 'cad56fcc-be02-43ea-a1a6-1d17f437acf7',
+    zoomRange: [0, 22],
+    opacity: 100,
+    visibility: 'visible',
+    source: {
+      dataURL: config.opensearchVectorTileDataUrl,
+    },
+    style: {
+      styleURL: config.opensearchVectorTileStyleUrl,
+    },
+  },
+  {
+    name: 'Cancelled flights',
+    description: 'Shows cancelled flights',
+    type: 'documents',
+    id: 'f3ae28ce-2494-4e50-ae31-4603cfcbfd7d',
+    zoomRange: [2, 22],
```

Comment:
How about make the example of document layer to be visible when user initially open it at zoom 0, like `[0, 5]`? IMO, as a demo example, it should be visualization friendly when at first glance.

Reply:
IMO, We should have use case why some layers can be invisible from beginning. User might not notice if everything is visible starting from 0.

Reply:
Agree, so that we can show layer is invisible after like zoom 5. I think the initially display on demo example is important for new users to show the main features. But yeah, any choice should be good.

Reply:
Lets make it between [0 - 14] like junqiu suggested.

Reply:
Sure, I will set the zoom range to [0-14] in the sample object of actual default map while checking it into the repo. As this is a dummy sample object just to validate the changes. For now, I'm leaving it as it is.